Automattic/wp-calypso
# Pricing page rework: Rename components to remove confusion

#### Proposed Changes

This PR renames
- `ProductFilter` component to `ViewFilter`
-  `Products` component to `ItemsList`
- `filterType/setFilterType` to `currentView/setCurrentView`

#### Testing Instructions



* Boot up this PR 
    * Run `git fetch && git checkout update/jp-pricing-rework/rename-components`
    * Run `yarn start-jetpack-cloud`
    * Goto [http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1](http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1)
* Or click on Jetpack Cloud live link below
    * Goto `/pricing?flags=jetpack/pricing-page-rework-v1`
* Confirm that everything looks same as before

#### Pre-merge Checklist






Slack conversation: p1661432728942369-slack-C03RCHNVC0H
